main/php_syslog.h

```diff
@@ -30,26 +32,12 @@
 #endif
 #endif
 
-/*
- * The SCO OpenServer 5 Development System (not the UDK)
- * defines syslog to std_syslog.
- */
-
-#ifdef syslog
-
-#ifdef HAVE_STD_SYSLOG
-#define php_syslog std_syslog
-#endif
-
-#undef syslog
+BEGIN_EXTERN_C()
+PHPAPI void php_syslog(int, const char *message);
```

Comment:
Done.  Although at this point, the string is being traversed 4 times by my count... so not very efficient.